examples/quantization_w8a8_fp8/README.md

```diff
@@ -22,122 +20,102 @@ The example includes an end-to-end script for applying the quantization algorith
+
+model = SparseAutoModelForCausalLM.from_pretrained(MODEL_ID,
+                                                   device_map="auto", 
+                                                   torch_dtype="auto")
 tokenizer = AutoTokenizer.from_pretrained(MODEL_ID)
 ```
 
-### 2) Prepare Calibration Data
-
-Prepare the calibration data. When quantizing activations of a model to `fp8`, we need some sample data to estimate the activation scales. As a result, it is very useful to use calibration data that closely matches the type of data used in deployment. If you have fine-tuned a model, using a sample of your training data is a good idea.
-
-In our case, we are quantizing an Instruction tuned generic model, so we will use the `ultrachat` dataset. Some best practices include:
-* 512 samples is a good place to start (increase if accuracy drops)
-* 2048 sequence length is a good place to start
-* Use the chat template or instrucion template that the model is trained with
-
-```python
-from datasets import load_dataset
-
-NUM_CALIBRATION_SAMPLES=512
-MAX_SEQUENCE_LENGTH=2048
-
-# Load dataset.
-ds = load_dataset("HuggingFaceH4/ultrachat_200k", split="train_sft")
-ds = ds.shuffle(seed=42).select(range(NUM_CALIBRATION_SAMPLES))
+### 2) Apply Quantization
 
-# Preprocess the data into the format the model is trained with.
-def preprocess(example):
-    return {"text": tokenizer.apply_chat_template(example["messages"], tokenize=False,)}
-ds = ds.map(preprocess)
+For `fp8` quantization, we can recover accuracy with simple PTQ quantization.
 
-# Tokenize the data (be careful with bos tokens - we need add_special_tokens=False since the chat_template already added it).
-def tokenize(sample):
-    return tokenizer(sample["text"], padding=False, max_length=MAX_SEQUENCE_LENGTH, truncation=True, add_special_tokens=False)
-ds = ds.map(tokenize, remove_columns=ds.column_names)
-```
-
-### 3) Apply Quantization
+We recommend targeting all `Linear` layers using the `FP8_Dynamic` scheme, which uses:
+- Static, per-channel quantization on the weights
+- Dynamic, per-token quantization on the activations
 
-With the dataset ready, we will now apply quantization.
-
-We first select the quantization algorithm. In our case, we will apply the default recipe for `fp8` (which uses static-per-tensor weights and static-per-tensor activations) to all linear layers.
-> See the `Recipes` documentation for more information on making complex recipes
+Since simple PTQ does not require data for weight quantization and the activations are quantized dynamically, we do not need any calibration data for this quantization flow.
 
 ```python
 from llmcompressor.transformers import oneshot
 from llmcompressor.modifiers.quantization import QuantizationModifier
 
 # Configure the quantization algorithm to run.
-recipe = QuantizationModifier(targets="Linear", scheme="FP8", ignore=["lm_head"])
+recipe = QuantizationModifier(targets="Linear", 
+                              scheme="FP8_Dynamic",
```

Comment:
Is it fine to not use all caps? I thought the scheme was `FP8_DYNAMIC`

Reply:
it seems to be working with `FP8_Dynamic`, but I can adjust it